test_conformance/images/kernel_read_write/test_cl_ext_image_from_buffer.cpp

```diff
@@ -79,77 +79,69 @@ int image2d_from_buffer_positive(cl_device_id device, cl_context context,
         return TEST_SKIPPED_ITSELF;
     }
 
-    std::vector<cl_mem_object_type> imageTypes{
-        CL_MEM_OBJECT_IMAGE1D,       CL_MEM_OBJECT_IMAGE2D,
-        CL_MEM_OBJECT_IMAGE3D,       CL_MEM_OBJECT_IMAGE1D_BUFFER,
-        CL_MEM_OBJECT_IMAGE1D_ARRAY, CL_MEM_OBJECT_IMAGE2D_ARRAY
-    };
+    cl_mem_object_type imageType = CL_MEM_OBJECT_IMAGE2D;
```

Comment:
Minor: since this value never changes I think it help understanding if it were "const".

Or, even better, just remove it and pass around CL_MEM_OBJECT_IMAGE2D explicitly.

Reply:
will do thank you